editor/src/components/canvas/controls/select-mode-control-container.tsx

```diff
@@ -494,8 +496,14 @@ export class SelectModeControlContainer extends React.Component<
   render() {
     const cmdPressed = this.props.keysPressed['cmd'] || false
     const allElementsDirectlySelectable = cmdPressed && !this.props.isDragging
-    const roots = MetadataUtils.getAllStoryboardChildrenPathsScenesOnly(
-      this.props.componentMetadata,
+    const storyboardChildren = MetadataUtils.getAllStoryboardChildren(this.props.componentMetadata)
+    const roots = mapDropNulls(
```

Comment:
Should the logic here and in `insert-mode-control-container.tsx` be refactored out as they look identical?